.gitattributes

```diff
@@ -0,0 +1,6 @@
+*.config text eol=lf
```

Comment:
I was under the impression that this was [only an issue on VMs](https://github.com/jenkins-infra/helpdesk/issues/3865#issuecomment-1854436671) not the containers?

If not this will be needed to be rolled out to every single project which seems extreemly sub-optimal.  
Somone should either force the Jenkins agents to use non default settings once for everyone, or fix spotless?

Reply:
> I was under the impression that this was https://github.com/jenkins-infra/helpdesk/issues/3865#issuecomment-1854436671 not the containers?

That's correct.  I ran the job with a Windows VM because Windows container builds are currently blocked due to an Azure Container outage.  That container outage is assumed that it will last through the weekend.

I think that the short term solution is to fix the Windows VM agents so that they don't transform line endings on checkout.  That is probably also the preferred long term solution so that we don't need to add a `.gitattributes` file to every repository.

Reply:
> That's correct. I ran the job with a Windows VM because Windows container

Ahh this was a replay with a changed Jenkinsfile then!  (NB I think you could have also run `git config set core.eol lf` after checkout and before running maven) and then not need the `.gitattributes` changes.  (which would also validate if the issue if from some lack of core.autocrlf / other setting in the VMs